cves/kernel/CVE-2015-8967.yml

```diff
@@ -55,7 +55,14 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A security vulnerability was located on the 'arch/arm64/kernel/sys.c' file prior
```

Comment:
The description should not include version types.

Reply:
Resolved. Thanks for pointing that out!